<launch>

    <!-- VSLAM Param -->
    <rosparam file="$(find vslam)/config/vslam_param.yaml"/>
    
    <!-- Camera Calibration Param -->
    <rosparam file="$(find vslam)/config/vslam_param.yaml"/>

    <!-- img path -->
    <param name="imgPath" type="str" value="$(find vslam)/data/image.jpg"/>
    <param name="imgPath2" type="str" value="$(find vslam)/data/image2.jpg"/>
    
    <!-- Realsense camera launch file -->
    <!-- https://robots.uc3m.es/installation-guides/install-realsense2.html -->
    <include file="$(find realsense2_camera)/launch/rs_d435.launch"/>

    <node name="feature_detection" pkg="vslam" type="feature_detection_node" output="screen"/>

</launch>